integration-cli/docker_test_vars.go

```diff
@@ -26,28 +25,15 @@ var (
 	workingDirectory string
 )
 
-func binarySearchCommand() *exec.Cmd {
-	if runtime.GOOS == "windows" {
-		// Windows where.exe is included since Windows Server 2003. It accepts
-		// wildcards, which we use here to match the development builds binary
-		// names (such as docker-$VERSION.exe).
-		return exec.Command("where.exe", "docker*.exe")
-	}
-	return exec.Command("which", "docker")
-}
-
 func init() {
 	if dockerBin := os.Getenv("DOCKER_BINARY"); dockerBin != "" {
 		dockerBinary = dockerBin
-	} else {
-		whichCmd := binarySearchCommand()
-		out, _, err := runCommandWithOutput(whichCmd)
-		if err == nil {
-			dockerBinary = stripTrailingCharacters(out)
-		} else {
-			fmt.Printf("ERROR: couldn't resolve full path to the Docker binary (%v)", err)
-			os.Exit(1)
-		}
+	}
```

Comment:
I have no idea what's going on here, but logic is changed. Now data from `LookPath` will override `DOCKER_BINARY`. Maybe that's your plan...